tests/integration/test_analysisd/test_signature_id/data/rules_samples/valid_signature_id_rules.xml

```diff
@@ -0,0 +1,11 @@
+
+<group name="syscollector,">
+
+    <rule id="969696" level="3" >
+        <description>Initial rule that works as parent of rule with invalid if_sid</description>
+    </rule>
+    <rule id="969697" level="3" >
+        <if_sid>969696, 969698</if_sid>
+        <description>Custom Rule with valid and invalid if_sid</description>
+    </rule>
+</group>
```

Comment:
I don't see it used in any case or test.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3649/commits/e4b77a1f8ea90438672bb86d62481c915d1d771b